docs/examples/infinite-scroll.md

```diff
@@ -7,10 +7,8 @@ In this example, we use a [third party library](https://github.com/Akryum/vue-ob
 
 For this to work we do 2 things:
 
-1. We tell the Results component to `stack` results
-1. We introduce a "dummy" div at the bottom that will increase the page number when it becomes visible
-
-**Info: because the example below is iframed, loading of results is a bit slowish. We recommend you take a [look here if you want optimal performances](https://s.codepen.io/rayrutjes/debug/dWBMje/XxMVwDjEywor).**
```

Comment:
It felt weird reading a sentence that said "Excuse us our super search is super slow and we do not really know why but this is an iframe ". (that's how I read it). Our search could totally be in an iframe.